api/management/commands/ingest_appeals.py

```diff
@@ -63,10 +63,12 @@ def handle(self, *args, **options):
                 country = country.first()
 
             appeals = [a for a in r['Details'] if a['APD_code'] not in aids]
+            atypes = {66: AppealType.DREF, 64: AppealType.APPEAL}
```

Comment:
Noting that the last type is 'international and it will be code `1537`. I will add this in.